scripts/nginx-switch-port.sh

```
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+NGINX_CONF="/etc/nginx/sites-available/api.dev.debate-timer.com"
+BACKUP_CONF="/etc/nginx/sites-available/api.dev.debate-timer.com.backup"
+TEMP_CONF="/tmp/api.dev.debate-timer.com.tmp"
+
+log() {
+    local timestamp=$(date '+%Y-%m-%d %H:%M:%S')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)